posthog/templates/email/second_ingestion_reminder.html

```diff
@@ -5,7 +5,7 @@ <h3>Hi, {{ first_name }}!</h3>
     <p>
         If you're having trouble getting started, you can
         <a
-            href="https://posthog.com/signup/self-host/get-in-touch?{{ utm_tags }}#demo"
+            href="https://posthog.com/book-a-demo?{{ utm_tags }}#demo"
```

Comment:
Small nit, it seems like the `#demo` target doesn't do anything, so probably best to remove, but not totally necessary if you're not sure.